### PR TITLE
Load CSG1

### PR DESCRIPTION
**Pull request by IntelOrca**
Merge requested by @Gymnasiast 

**Includes:**
* Loading RCT1's `csg1*.dat` graphics if RCT1 path is specified
* Small refactor of landscape sprites

**Note:**
Loading csg graphics adds about 60MB RAM and we won't want to load it like this anyway - but it's convenient for further experiments with those graphics.

### CODE REVIEW

**Comment by janisozaur** [comment]
Nice feature to have 👍 
If only I found my RCT CD…

File: src/openrct2/sprites.h [hunk 1]
```diff
@@ -746,6 +751,20 @@ enum {
 	SPR_G2_SORT = SPR_G2_BEGIN + 77,
 	SPR_G2_COPY = SPR_G2_BEGIN + 78,
 	SPR_G2_PASTE = SPR_G2_BEGIN + 79,
+
+	SPR_CSG_BEGIN = 393216,
```
Comment (marijnvdwerf) [inline src/openrct2/sprites.h]: What is this number based on?

Comment (IntelOrca) [inline src/openrct2/sprites.h]: Its so significant that it doesn't require a constant or comment.

Comment (Gymnasiast) [inline src/openrct2/sprites.h]: Well, apparently it does need a comment, as it isn't obvious to Marijn or to me.

Comment (IntelOrca) [inline src/openrct2/sprites.h]: Its a round hex number of the last possible range of image ID values that is large enough to fit all csg sprites.

File: src/openrct2/sprites.h [hunk 2]
```diff
@@ -752,8 +752,10 @@ enum {
 	SPR_G2_COPY = SPR_G2_BEGIN + 78,
 	SPR_G2_PASTE = SPR_G2_BEGIN + 79,
 
+	// 0x60000, chosen because it's a round hex number
+	// of the last possible range of image ID values that is large enough to fit all csg1 sprites.
 	SPR_CSG_BEGIN = 393216,
-
+
```
Comment (IntelOrca) [inline src/openrct2/sprites.h]: ?

Comment (Gymnasiast) [inline src/openrct2/sprites.h]: Typo.

Comment (Gymnasiast) [inline src/openrct2/sprites.h]: Has been removed in next commit.

**Pull request merged**